### DISCUSSION TITLE
How to jump from current window to Neotree

### DISCUSSION THREAD

**Discussion by alewis001** [LazyVim/LazyVim#2488] [Q&A] [answered]
This may be a stupid question... 😄 

The keymappings for Neotree all appear to work as a toggle rather than an "open and focus". Have I just missed a keymap for the "open and focus" behaviour or is there just no default for that in LazyVim? Or, do I just have the wrong expectations for how to properly use Neotree? 

I realise I can just move between the windows with, in my case, `ctri+h,j,k,l`. I'd prefer to jump from whatever window I'm in straight to Neotree.

The `:NeoTree dir=./` command appears to have the "open and focus" behaviour that I'd want so I can adjust keymaps, I just didn't want to get into that if there is already a way to do it or I was just not using Neotree correctly.

Cheers

**Comment by kobzar** [comment, marked as answer]
-- Neo Tree
map("n", "<leader>o", "<cmd>Neotree focus<cr>", { desc = "NeoTree focus" })

**Reply by alewis001** [reply]
Ok, great, thanks!

I'm now trying out the floating mode to see whether that works for me.

**Reply by olaberglund** [reply]
And for anyone seeing this later, remember to surround the command with the correct tags: `"<cmd>Neotree focus<cr>"`
